### ISSUE TITLE
Gray text on gray background on conversation dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#205]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1.when you try to talk to some one you can not see the words
2.
3.

What is the expected output? What do you see instead?
You should be able to read them but can't because of the color if the words is 
same as back ground

What version of the product are you using? On what device?
Using the latest version I have a droid x with android 2.3.3 on Verizon network

Please provide any additional information below.
This didn't start till I updated to android 2.3.3 

```

Original issue reported on code.google.com by `buck...@gmail.com` on 2 Jul 2011 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
who were you trying to talk to
```

Original comment by `dope...@gmail.com` on 2 Jul 2011 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Anyone, the background of the last thing the person says is the same color as 
the words.  I also just recently updated to Gingerbread (2.3.3) on my Droid X.  
Screenshot at http://tinypic.com/r/2zjdw11/7

```

Original comment by `enda...@gmail.com` on 4 Jul 2011 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the report. I have read other reports from people doing that 
update having the same problem with other applications. It seems the 
manufacturer (Motorola?) included a theme update in there, which makes a lot of 
applications show up this way.

Unfortunately, I have no way to reproduce this since the error does not show in 
the emulator. Anyway, I now added code that forces a black background on the 
conversation list. Hopefully this should solve the issue.

Thanks for reporting it!

Should be fixed in r121. Marking this as ready for the next release.
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 3:54
- Changed title: **Gray text on gray background on conversation dialog**
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
This is apparently not resolved yet, as some user testing on an actual Motorola 
Droid X still exhibits the problem. Reopening.
```

Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hmm, I wonder if the fixes done for issue 242 could affect his also?
```

Original comment by `oskar.wi...@gmail.com` on 9 Sep 2011 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
Oskar,

If you need me to test this again on my DROIDX,  then please send me an exe
```

Original comment by `jgemmajr@gmail.com` on 9 Sep 2011 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
I have this same issue on a Droid 2 running 2.3.3

Something that the original complaint neglects:  The text has a grey background 
because it appears to be selected or highlighted.  If you touch the text, or a 
previous line of text, the selection focus changes and the highlight goes away.

Hope this helps.
```

Original comment by `icanhazn...@gmail.com` on 27 Sep 2011 at 7:29

**Comment by GoogleCodeExporter** [comment]
```
In the alpha version of 0.6.10 this issue has been cleaned-up slightly.   The 
gray background is still very much an issue when a radio button is present.  
When no radio button appears the background color is black.
All other buttons still appear with a gray background as well.

DROID x 2.3.3
```

Original comment by `jgemmajr@gmail.com` on 9 Oct 2011 at 1:43

**Comment by GoogleCodeExporter** [comment]
```
I get this effect when I touch my screen in the About/help section of the app 
but not anywhere else.

Running CM 7.1 Android version 2.3.4
```

Original comment by `sdeva...@gmail.com` on 9 Oct 2011 at 4:55

**Comment by GoogleCodeExporter** [comment]
```
The most game-breaking bug of the gray text on gray background is now solved 
with r173.

For reference, the changes needed are summarized in this article:
http://community.developer.motorola.com/t5/MOTODEV-Blog/Why-Does-My-ListView-Loo
k-Different/ba-p/17462

Marking this issue as ready for v0.6.10.
```

Original comment by `oskar.wi...@gmail.com` on 9 Oct 2011 at 11:34
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
